frontend/src/hooks/logo-hooks.ts

```diff
@@ -14,7 +14,7 @@ export function useLogo(s3Key?: string): UseLogoHook {
     if (s3Key) {
       try {
         setLoading(true);
-        const data = await StorageService.downloadLogo(s3Key);
+        const data = await StorageService.downloadLogo(s3Key, () => {});
```

Comment:
Good find, I'm wondering how this was working before?

Reply:
It seems like TypeScript (or JavaScript) function parameters are defaulted to undefined according to [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters). In our case, the passed in function is called when downloadFile encounters an error. I'll change the dummy function to the translation function.